plugin/plugin_settings.py

```diff
@@ -117,7 +117,7 @@ def __update_settings_from_project_if_needed(self):
         self.__load_vars_from_settings(settings_handle, Settings.PREFIXES)
         log.debug(" All overrides applied.")
 
-    def __load_vars_from_settings(self, settings_handle, prefixes=[""]):
+    def __load_vars_from_settings(self, settings_handle, prefixes=None):
```

Comment:
Not sure I understand reason for this change. Care to explain? If there is no specific reason then I would argue that previous was clearer.